src/test/java/de/mediathekview/mserver/crawler/CrawlerManagerFTPTest.java

```diff
@@ -59,7 +57,7 @@ public CrawlerManagerFTPTest(final String aFilmlistPath, final FilmlistFormats a
 
   @Parameterized.Parameters(name = "Test {index} Filmlist FTP upload for {0} with {1}")
   public static Collection<Object[]> data() {
-    return Arrays.asList(new Object[][] {{"TestFilmlistNewJson.json", FilmlistFormats.JSON},
+    return Arrays.asList(new Object[][]{{"TestFilmlistNewJson.json", FilmlistFormats.JSON},
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this array creation and simply pass the elements. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS3878)